build/checkOrphanedImages.js

```diff
@@ -0,0 +1,91 @@
+const path     = require('path');
+const fs       = require('fs');
+const common   = require('../scripts/common');
+const {
+  prompt,
+  flatten,
+  fetchFiles } = require('./utils/cli');
+
+const ignored_files            = [];
+const ignored_image_extensions = [
+    'DS_Store', 'zip',
+];
+
+const ignored_language_directories = [
+    'en', 'de', 'es', 'fr', 'id', 'it', 'pl', 'pt', 'ru', 'th', 'vi', 'zh_cn', 'zh_tw',
+];
+
+module.exports = function(grunt) {
+    prompt('Checking for images with bad naming...');
+    grunt.initConfig({
+        checkOrphanedImages: {},
+    });
+
+    grunt.registerTask('check_orphaned_images', 'Check orphaned images', () => {
+        const default_path                = grunt.option('all') ? '' : 'app_2';
```

Comment:
@easteregg it would be use full if there were a switch (e.g. --delete) to remove orphaned images too.